README.rst

```diff
@@ -65,9 +72,11 @@ docker-compose.yaml::yaml
         chadburn.job-exec.rss2email.schedule: "@every 5m"
         chadburn.job-exec.rss2email.command: "r2e run"
 
-The example docker-compose snippet sets up chadburn (or ofelia) job scheduler to run r2e periodically. For the labels to be picked up, it is required that the container keeps running which is ensured with the entrypoint. Python's Event().wait() was used as there is no shell and this method should use less cpu than sleeping or looping (with sleep). You can of course abandon this and just call r2e from any other job scheduler as this entrypoint only circumvents chadburn/ofelia limitations.
+The example docker-compose snippet sets up chadburn (or ofelia) job scheduler to run r2e periodically. 
+
+Some background for entrypoint override: For the labels to be picked up, it is required that the container keeps running which is ensured with the entrypoint. Python's Event().wait() was used as there is no shell and this method should use less cpu than sleeping or looping (with sleep). You can of course abandon this and just call r2e from any other job scheduler as this entrypoint only circumvents chadburn/ofelia limitations.
 
-You need to run `chown 65532:65532 config data` to have the correct ownership of files for this rootless docker image.
+You need to run ``chown 65532:65532 config data`` to have the correct ownership of files for this rootless docker image.
```

Comment:
Why the double quotes here and in other places?

Reply:
@auouymous 
https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#inline-markup

Reply:
Saw it in other places in the README used to do an inline literal. So same as the above, this was my first time using RST, so I am not sure how better to do it